features/helper-plugin.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Manipulate a project using Tuist with a helper plugin and be able to import it in all supported manifest types.
+
+  Scenario: The project is an iOS application (app_with_git_helper_plugin)
+    Given that tuist is available
+    And I have a working directory
+    Then I copy the fixture app_with_git_helper_plugin into the working directory
+    Then tuist generates the project
+    Then I should be able to build for iOS the scheme App
```

Comment:
I don't think this test is adding much value so I'd rather add it when we have logic built upon the plugins system. Any of those tests will implicitly test that the configuration can be loaded and parsed.

Reply:
Understood, I'll remove this fixture for now.